chapter_12/3_Reconstruction.tex

```diff
@@ -0,0 +1,66 @@
+%Contributor: Ben Brimacombe
+\section{Geometry and Homology Inference}
+The nerve theorem is a powerful result which we can use for several more analyses of the topology of data. The first of these techniques covers $X_n$ by a union of balls of fixed radius centered on the $x_i$. This construction has 
+’s. Under some regularity
+assumptions on M, one can relate the topology of this union of balls to the one of M;
+2. From a practical and algorithmic perspective, topological features of M are inferred from
```

Comment:
What is this 2.? This paragraph seems to have some typos in it which make it hard to understand.